interface-spec.html

```diff
@@ -146,8 +146,18 @@ <h3><dfn>Term</dfn> interface</h3>
       <dfn>value</dfn> is refined by each interface which extends Term.
     </p>
     <p>
-      <dfn>equals()</dfn> returns <code>true</code> if and only if <code>other</code> has the same
-      <code>termType</code> and the same contents (as defined by concrete subclasses).
+      <dfn id="term-equals">equals()</dfn> returns <code>true</code>
+      when called with parameter <code>other</code>
+      on an object <code>term</code> if and only if
+      all of the conditions below hold:
+    </p>
+    <ul>
+      <li><code>other</code> is <em>neither</em> <code>null</code> nor <code>undefined</code>;</li>
+      <li><code>term.termType</code> is the same string as <code>other.termType</code>;</li>
+      <li><code>other</code> follows the additional constraints followed by the most specific <code>Term</code> interface implemented by <code>term</code> (as detailed below);</li>
```

Comment:
`followed by the most specific <code>Term</code>`

This text makes it seem that multiple interfaces could be implemented, which shouldn't be possible. Should this be _"the more specific Term"_ instead?

Reply:
I must admit that I don't understand what this last item means about the more specific Term interface?

Reply:
It means NamedNode/Literal/… will rephrase, probably enumerate.